src/examples/DateTimePickerExamplePage.tsx

```diff
@@ -0,0 +1,39 @@
+'use strict';
+import {StyleSheet, View, Text, Platform} from 'react-native';
+import React, {useState} from 'react';
+import DateTimePicker from '@react-native-community/datetimepicker';
+
+const styles = StyleSheet.create({
+  container: {
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+});
+
+export const DateTimePickerExamplePage: React.FunctionComponent<{}> = () => {
+    const [date, setDate] = useState(new Date());
+    const [time, setTime] = useState(undefined);
+    
+  return (
+    <View style={styles.container}>
+      <Text>Sample Date Picker</Text>
+      <DateTimePicker
+        value={date}
+        mode="date"
+        display="default"
+        maximumDate={new Date(2050, 12, 31)}
+        minimumDate={new Date(1950, 1, 1)}
+        dayOfWeekFormat={'{dayofweek.abbreviated(3)}'}
+      />
+      <Text>Sample Time Picker</Text>
+      <DateTimePicker
+        mode="time"
+        value={time}
+        style={{width: 300, opacity: 1, height: 30}}
+        is24Hour={false}
```

Comment:
consider binding this to a toggleswitch's state?

Reply:
Good question... how much interactivity should there be on each page. If the reference is XAML Controls Gallery these aren't meant to be comprehensive test pages like we might have for the internal tests. Looking at the XAML Controls Gallery page for DatePicker and TimePicker there's no options to toggle the UI settings, but there are multiple instances of the control (with markup) to showcase its options.

![image](https://user-images.githubusercontent.com/26607885/96165724-d6d26b00-0ed1-11eb-8bc4-c8eb5da1f28f.png)

![image](https://user-images.githubusercontent.com/26607885/96165691-ccb06c80-0ed1-11eb-8e6b-6188261c2ff9.png)

Reply:
Let's think how many options we want to display. On the one hand it makes sense to make it rich, but then there're already the example app in the community module repos (maybe the way is to copy those samples here? they should work OOTB).